frontends/main/src/app-pages/HomePage/HomePage.tsx

```diff
@@ -52,25 +52,21 @@ const HomePage: React.FC = () => {
         <StyledContainer>
           <HeroSearch />
           <section>
-            <Suspense>
```

Comment:
No longer seeing the "useSearchParams() should be wrapped in a suspense boundary" error?